site/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "build": "bundle exec jekyll build",
+    "dev": "bundle exec jekyll serve",
+    "dev-sw": "npm run build && sw-precache --config=sw-precache-config.js && npm run dev",
```

Comment:
If we don't build first then the html & css hasn't been compiled or the assets moved. I tested by deleting the `_site` directory and running `sw-precache --config=sw-precache-config.js && npm run dev` which resulted in a service worker being generated but it not actually caching anything if that makes sense.